spring-ai-docs/src/main/antora/modules/ROOT/pages/concepts.adoc

```diff
@@ -115,7 +123,7 @@ On input, models convert words to tokens. On output, they convert tokens back to
 
 In English, one token roughly corresponds to 75% of a word. For reference, Shakespeare's complete works, totaling around 900,000 words, translates to approximately 1.2 million tokens.
 
-Perhaps more important is that Tokens = *`$`*.
+Perhaps more important is that Tokens = Money.
```

Comment:
This suggested change is for making the statement more international by removing currency specifics.